go/fn/example_kubeobject_test.go

```diff
@@ -12,59 +12,37 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package example
+package fn
 
 import (
-	"github.com/GoogleContainerTools/kpt-functions-sdk/go/fn"
 	corev1 "k8s.io/api/core/v1"
```

Comment:
Have you updated `go.mod`?
IIUC moving an example_test file that depends on k8s package, to the main module will introduce k8s package as dependencies in go.mod. This is something we want to avoid.

Reply:
ouch... Yeah I updated to include k8s.io/api, and I saw the comment (thought it is only for apimachinary)